### PR TITLE
#98 canned postgres

### PR DESCRIPTION
**Pull request by nikolayk812**
- no postgres client exposed to API
- less code

### CODE REVIEW

**Comment by nikolayk812** [comment]
Defintely inspired by @giorgioazzinnaro

File: canned/postgres/postgres.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	"fmt"
+	"github.com/docker/go-connections/nat"
+	"github.com/pkg/errors"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	image             = "postgres"
+	tag               = "9.6.15"
```
Comment (borgotta) [inline canned/postgres/postgres.go]: Don't you think that it will be better if we use some fresher versions by default? Like 12.1-alpine, for example.

**Comment by DerThorsten** [comment]
Will this be merged ? is there anything where I could help to get this merged?

**Comment by Streppel** [comment]
I second @DerThorsten . I can help too if needed.

**Pull request closed without merging**